catalog/core/catalog-core-urlresourcereader/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -30,7 +30,20 @@
         </set>
       </property>
   </bean>
-  
+
+    <bean id="redirectUrlReader" class="ddf.catalog.resource.impl.URLResourceReader">
+        <cm:managed-properties
+                persistent-id="ddf.catalog.resource.impl.URLResourceReader"
+                update-strategy="container-managed"/>
+        <argument ref="mimeTypeMapper"/>
+        <property name="rootResourceDirectories">
+            <set>
+                <value type="java.lang.String">${ddf.home}/data/products</value>
+            </set>
+        </property>
+        <property name="followRedirects" value="true"/>
```

Comment:
This is that secondary instance we created specifically for following redirects. It is exposed with a different id and shortname. This is what was needed to support things like the confluence source.